### PR TITLE
Add installing info

### PR DESCRIPTION
**Pull request by evilnick**
Pages added:

 - installing index (stub)
 - Installing from a snap (complete)
 - Installing with multipass (stub)
 - explanation/channels

### CODE REVIEW

File: docs/src/explanation/channels.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+### Which channel is right for me?
+
+Choosing the most appropriate channel for your needs depends on a number of
+factors. We can give some general guidance for the following cases:
+
+- **I want to always be on the latest stable version matching a specific upstream K8s
+release (recommended).** 
+
+Specify the release, for example: `--channel=1.25/stable`.
```
Comment (kwmonroe) [inline docs/src/explanation/channels.md:73]:
```suggestion
Specify the release, for example: `--channel=1.30/stable`.
```
Stop living in the past

File: docs/src/howto/install/snap.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+## What you'll need
+
+This guide assumes the following:
+
+- You are installing on Ubuntu 22.04 or later, **or** another OS which supports
+  snap packages (see [snapd support])
+- You have root or sudo access to the machine
+- You have an internet connection
+- The target machine has sufficient memory and disk space. To accommodate
+  workloads, we recommend a system with at least 20G of disk space and 4G of
```
Comment (kwmonroe) [inline docs/src/howto/install/snap.md:15]: 4G / 20G is fine for a basic install, but I think if we're gonna say *recommend*, we should bump those up. If something like 8G feels too heavy, perhaps emphasize the **at least** part and wrap it in a `<blink>` tag or something.

FWIW, we're upping the charmed k8s k-c-p recommended specs to 8G / 64G in https://github.com/charmed-kubernetes/bundle/pull/892.

File: docs/src/howto/install/snap.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  memory.
+
+```{note}
+If you cannot meet these requirements, please see the [Installing] page for alternative options.
```
Comment (kwmonroe) [inline docs/src/howto/install/snap.md:19]:
```suggestion
If you cannot meet these requirements, please see the [Installing][installing] page for alternative options.
```

**Review by kwmonroe — CHANGES_REQUESTED** [review]
Lookin good!  There's a couple legit changes plus a couple suggestions for your consideration.

File: docs/src/explanation/channels.md (hunk 1 above)
Comment (evilnick) [reply docs/src/explanation/channels.md:73]: It was nicer then.

File: docs/src/howto/install/snap.md (hunk 3 above)
Comment (evilnick) [reply docs/src/howto/install/snap.md:19]: I'm pretty sure this link works - I think it just doesn't render properly in GH

Comment (kwmonroe) [reply docs/src/howto/install/snap.md:19]: ack, gh preview didn't so i thought it was the different case. i trust you may have been right, but appreciate the update to the links section anyway :)

**Review by kwmonroe — APPROVED** [review]
:cake:

**Pull request merged**